adapters/tlsio_openssl.c

```diff
@@ -933,7 +1010,14 @@ static int create_openssl_instance(TLS_IO_INSTANCE* tlsInstance)
         log_ERR_get_error("Failed allocating OpenSSL context.");
         result = __FAILURE__;
     }
-    else if (add_certificate_to_store(tlsInstance, tlsInstance->certificate) != 0)
+    else if (load_system_store(tlsInstance) != 0)
```

Comment:
do we want to break? or try to load the certificate using other methods (ifs) below?